source/extensions/filters/http/ext_proc/ext_proc.h

```diff
@@ -283,6 +283,16 @@ class Filter : public Logger::Loggable<Logger::Id::ext_proc>,
 
   void sendTrailers(ProcessorState& state, const Http::HeaderMap& trailers);
 
+  Http::LocalErrorStatus onLocalReply(const LocalReplyData&) override {
+
+    if (Runtime::runtimeFeatureEnabled(
+            "envoy.reloadable_features.ext_proc_disable_response_processing_on_local_reply")) {
+      ENVOY_LOG(trace, "local reply; will skip ext_proc response processing requests");
+      encoding_state_.inLocalReply(true);
```

Comment:
Recommend directly set below here.  If everybody agrees to skip ext_proc processing for local replies, you do not need the runtime flag.
 processing_complete_ = true;

With that , you can remove those changes  in encoderHeaders()/encodeData()/encodeTrailers()